WordPress-Core/ruleset.xml

```diff
@@ -217,14 +217,11 @@
 		<properties>
 			<property name="requiredSpacesAfterOpen" value="1"/>
 			<property name="requiredSpacesBeforeClose" value="1"/>
+
+			<!-- ... and for multi-line function calls, there should only be one parameter per line. -->
+			<property name="allowMultipleArguments" value="false" />
```

Comment:
Unit tests are [complaining](https://travis-ci.org/WordPress-Coding-Standards/WordPress-Coding-Standards/jobs/417092571) about the space at end of this tag. 🙃

Reply:
Thanks. Fixed up now.